Source/WebKit/Platform/cocoa/ExtensionCapabilityGrant.mm

```diff
@@ -35,14 +35,21 @@
 
 namespace WebKit {
 
-static void platformInvalidate(id<BEProcessCapabilityGrant> platformGrant)
+static void platformInvalidate(PlatformGrant platformGrant)
```

Comment:
This is a variant containing RetainPtr, it should not be passed by value. Maybe an r-value would be appropriate here. If not, a const reference.

Reply:
Will fix!